### PR TITLE
Feature/ub linux installation

### PR DESCRIPTION
**Pull request by ubozov**
Resolves Privatix/privatix#44.

### CODE REVIEW

File: inst/openvpn/misc_darwin.go [hunk 1]
```diff
@@ -11,3 +17,70 @@ func serviceName(prefix, path string) string {
...
+		Port:   port,
+	}
+	if err := templ.Execute(file, &d); err != nil {
+		return err
+	}
+
+	return exec.Command("launchctl", "load", daemonPath(name)).Run()
+}
+
+var daemonTemplate = `<?xml version="1.0" encoding="UTF-8"?>
```
Comment (drew2a) [inline inst/openvpn/misc_darwin.go]: May be it is better to extract this xml to the xml file.

File: inst/openvpn/misc_linux.go [hunk 2]
```diff
@@ -11,3 +17,64 @@ func serviceName(prefix, path string) string {
...
+		Server: server,
+	}
+	if err := templ.Execute(file, &d); err != nil {
+		return err
+	}
+
+	return exec.Command("systemctl", "enable", daemonPath(name)).Run()
+}
+
+var daemonTemplate = `[Unit]
```
Comment (drew2a) [inline inst/openvpn/misc_linux.go]: The same. May be it is better to extract.

File: inst/openvpn/misc_darwin.go (hunk 1 above)
Comment (ubozov) [reply inst/openvpn/misc_darwin.go]: updated

File: inst/openvpn/misc_linux.go (hunk 2 above)
Comment (ubozov) [reply inst/openvpn/misc_linux.go]: updated

**Pull request merged**